python3/totoro_agent/brain/enemy_tracker.py

```diff
@@ -5,21 +5,41 @@
 
 
 class EnemyTracker:
-    def __init__(self):
-        self.enemy_pos = [0, 0]
-
-    def update(self, game_state):
-        # TODO Yuv
-        # own position
-        # own health
-        # own ammo
-        # own diameter
-        # own invulnerable
-
-        # enemy ammo
-        # enemy position
-        # enemy health
-        # enemy diameter
-        # enemy is invulnerable
-
-        pass
+	def __init__(self):
+		#Enemy Tracker. Using enemy_ammo etc. even though it's THIS GUYS AMMO. we're not doubling reverse card. Enemy is OUR enemy. Global english, not relative.
+		self.enemy_pos = [0, 0] 
+		self.enemy_ammo = 3
+		self.enemy_health = 3
+		self.enemy_blast_radius = 2 # -> Not checking # of powerups, so just increase this by 1 whenever we pick up a powerup.
+		self.enemy_invulnerable_until = 0 # Get better var name, number is inclusive.
+
+
+	def update(self, game_state):
+		# TODO Yuv
+		self.player_id = str(game_state["connection"]["agent_number"])
+		# own position
+		# own health
+		# own ammo
+		# own diameter
+		# own invulnerable (we don't worry about our own stuff -- this'll be easily reflected)
+
+
+		self.enemy_id = str((game_state["connection"]["agent_number"]+1)%2) 
+
+		print(self.enemy_id, self.player_id)
+		self.enemy_ammo = game_state["agent_state"][self.enemy_id]["inventory"]["bombs"]
```

Comment:
Can first retrieve the enemy info and assign it to a variable and then retrieve properties from that assigned object. E.g.

```
enemy_state = game_state["agent_state"][self.enemy_id]
self.enemy_ammo = enemy_state["inventory"]["bombs"]
self.enemy_pos = enemy_state["coordinates"]
...
```